### ISSUE TITLE
Current buildable version of this?

### ISSUE THREAD

**Issue by kingjamez** [bastibl/gnuradio-android#18]
Building the docker doesn't work, and just getting the git clone doesn't work because of old commits as requirements.

Is anyone aware of a gnuradio on Andriod that is maintained and working?

**Comment by jrajason** [comment]
Yeah, but its messy. Once I had it running, I ran into a wall with converting the sample flowgraphs for android studio. Bastibl offers a couple of working flowgraphs but you have to really have a deep knowledge of the conversion to modify those sample flowgraphs. I fixed a good chunk of the errors and posted it to here: https://github.com/jrajason/OGGR

It does actually work, but conversion work is perplexing to me.

I'm not a GNURadio guy, but I went down the SCOPY version/path that Adrian Suciu made. Its a rabbit hole of incredible frustration and poorly managed. I found a way to load the docker and it didn't work build at all and I used the latest repo that was updated because the management and commits on there are not maintained at all.

My advice is to try and use this/BASTIBL's work or Tom Rondeau's work, if you have less need for some of BASTIBL's new functionality.

**Comment by kingjamez** [comment]
Really appreciate the tips. Thank you! I'll have a go at it and make it available if I have any luck.

**Comment by bastibl** [comment]
I only changed the URL of the openssl repo (it looks like they made their dev repo private) and docker builds without problems. Not sure what issues you are facing.

https://github.com/bastibl/gnuradio-android/commit/133efad883a96ad73b187270673d36f214875476

**Comment by jrajason** [comment]
I forgot what I changed but there were a couple of silly things that had to be updated in the dockerfile, I just forked your repo to get it working.
